automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/sam/SamApiSpec.scala

```diff
@@ -45,7 +48,7 @@ class SamApiSpec extends FreeSpec with Matchers with ScalaFutures {
     if (Sam.admin.doesUserExist(subjectId).getOrElse(false)) {
       Sam.admin.deleteUser(subjectId)
     }
-    Thurloe.keyValuePairs.deleteAll(subjectId)
+    thurloe.keyValuePairs.deleteAll(subjectId)
```

Comment:
Here and elsewhere that uses the new `Thurloe`... need to change the `AuthToken` import to:
```
import org.broadinstitute.dsde.workbench.auth.AuthToken
```